cmd/test.go

```diff
@@ -55,18 +49,16 @@ https://docs.snyk.io/products/snyk-infrastructure-as-code/custom-rules/getting-s
 		return nil
 	},
 	PostRun: func(cmd *cobra.Command, args []string) {
-		var path string
 		if len(args) == 0 {
```

Comment:
[This comment](https://github.com/snyk/snyk-iac-rules/pull/96/files#r757895197) also applies here.